packages/fuchsia_ctl/test/logger_test.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+void main() {
+  test('', () async {});
+}
```

Comment:
Maybe forgot to save before committing? :)

Reply:
Yep uploaded the version with the tests.